texastribune/styleguides
# Begin establishing systems to build off of for future styles

#### What's this PR do?

Begins establishing a typography system that we can use during style enhancements. Adds to style guide how icons will likely be included.
#### Why are we doing this? How does it help us?

Gets us thinking about enhancing styles with more flexible, modular, responsive systems, and experimenting with and building a base for what that will look like. By beginning now and giving ourselves more time to iterate, we'll get to a better place in the end.
#### How should this be manually tested?

Visit /newtribunestyles/. Check out the 'Typography' section - read through its text and look over its examples.
#### Are there performance implications? If adding JS can it be done async? Do images/CSS/JS have cache headers set?
#### What are the relevant tickets?

Part of Sprint task.
#### Next steps?

Keep iterating on these styles.
#### Smells?
#### TODOs:
#### Has the relevant documentation/wiki been updated?

This is the documentation for styles.
#### Technical debt note

Same
